From 52df04267562220920d9be652987f9a607d3cc77 Mon Sep 17 00:00:00 2001
From: evs-ch <evs-ch@users.noreply.github.com>
Date: Mon, 2 Dec 2019 12:28:24 +0100
Subject: [PATCH 2/3] Relax requirements a bit

Python packages on Arch are a bit more recent, so a somewhat more
relaxed list of requirements might be appropriate.
---
 requirements.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index b29db03..178c918 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,3 @@
-requests==2.22.0
-numpy==1.16.5
-setuptools==41.2.0
+requests>=2.22.0, <2.23
+numpy>=1.16.5, <1.18
+setuptools>=41.2.0, <42

From ca2133bdcef0a838dfed6db536a085388af6c0ac Mon Sep 17 00:00:00 2001
From: evs-ch <evs-ch@users.noreply.github.com>
Date: Fri, 6 Mar 2020 11:27:50 +0100
Subject: [PATCH 3/3] drop upper limits for requirements

---
 requirements.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index 178c918..1ec6135 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,3 @@
-requests>=2.22.0, <2.23
-numpy>=1.16.5, <1.18
-setuptools>=41.2.0, <42
+requests>=2.22.0
+numpy>=1.16.5
+setuptools>=41.2.0
